src/Microsoft.DotNet.Build.Tasks/PackageFiles/test-runtime/project.json

```diff
@@ -11,12 +11,14 @@
   "frameworks": {
     "dnxcore50": {
       "dependencies": {
+        "Microsoft.DotNet.xunit.performance.analysis.cli": "1.0.0-alpha-build0032",
+        "Microsoft.DotNet.xunit.performance.runner.cli": "1.0.0-alpha-build0032",
         "Microsoft.NETCore.Platforms": "1.0.1-rc2-24027",
         "Microsoft.NETCore.TestHost": "1.0.0-rc2-24027",
         "Microsoft.NETCore.Console": "1.0.0-rc2-24027",
         "xunit.console.netcore": "1.0.2-prerelease-00101"
       },
-      "imports": "portable-net45+win8"
+      "imports": ["portable-net45+win8", "netstandard1.3"]
```

Comment:
Thing to note here is this will affect all test builds in repos that take a dependency on Build.Tasks ( I believe this is from https://github.com/dotnet/buildtools/pull/488/files )